notifications_utils/version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "50.0.0"
```

Comment:
I changed the version to 50.0.1 so there's a conflict, but you can just go with 50.1.0 here 👍